waffle-compiler/test/flattener/flattener.ts

```diff
@@ -0,0 +1,34 @@
+import {expect} from 'chai';
+import fs from 'fs-extra';
+import {inputToConfig} from '../../src/config';
+import {flattenAndSave} from '../../src/flattener';
+
+const sourceDirectory = './test/flattener/testSource';
+const flattenOutputDirectory = './test/flattener/flattenFiles';
+const compilerVersion = 'v0.5.9+commit.e560f70d';
+const config = inputToConfig({
+  sourceDirectory,
+  flattenOutputDirectory,
+  compilerVersion
+});
+
+describe('flattening', () => {
+  after(async () => {
+    await fs.remove(flattenOutputDirectory);
+  });
+
+  it('should include the parent if the child file', async () => {
+    await flattenAndSave(config);
+    const file = fs.readFileSync('./test/flattener/flattenFiles/child.sol', 'utf8');
+    expect(file).includes('contract Parent');
+  });
+
+  it('should leave multiple solidity pragmas', async () => {
```

Comment:
The output file of a flattener generally should **not** include multiple solidity pragmas. This tests passes because they are left in `// Dependency file: ...` comments.

I'd leave the original pragma of each dependency file commented out in the flattener output file like this: 
```solidity
// Dependency file: openzeppelin-solidity/contracts/access/roles/PauserRole.sol
// pragma solidity ^0.5.0;

contract PauserRole {
    ...
}
```
...and then change the name and expect statement of this test to reflect that we are checking commented out pragmas.